HeterogeneousCore/AlpakaServices/plugins/BuildFile.xml

```diff
@@ -1,9 +1,6 @@
 <!-- alpaka-based portable plugins -->
 <library file="alpaka/*.cc" name="HeterogeneousCoreAlpakaServicesPlugins">
   <use name="alpaka"/>
-  <use name="FWCore/MessageLogger"/>
-  <use name="FWCore/ParameterSet"/>
-  <use name="FWCore/PluginManager"/>
```

Comment:
Thanks for the clean up.

Do you mind if I cherry pick your commit for this file into a different branch, and make a separate PR ?
I'm trying to keep all changes to the Alpaka-related code in a dedicate branch, to ease backports _etc_.

Reply:
Yes that makes sense. I'll revert the change to this file in this PR then!